person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, name = 'Unknown', age = 0, parent_permission: true)
+    @id = id
```

Comment:
- [ ] Kindly remove `id` from the list of parameters here as it can be auto-generated from the app, so we don't need it. I will show you how below.

- [ ] The `age` parameter is not an optional parameter so it should be declared this way, and you need to arrange your parameters to reflect this.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = rand(1...1000)  # generates a random integer between 1 and 1000 inclusive
    ...
  ```